cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -83,11 +88,12 @@ def test_single_vm_migration_power_state_tags_retirement(appliance, provider,
     assert migration_plan.wait_for_state("Successful")
 
     # check power state on migrated VM
-    rhv_prov = provider
-    migrated_vm = rhv_prov.mgmt.get_vm(src_vm.name)
-    assert power_state in migrated_vm.state
+    migrated_vm = get_migrated_vm(src_vm, provider)
+    vm_state = migrated_vm.state if provider.one_of(RHEVMProvider) else migrated_vm.mgmt.state
```

Comment:
Can't you just take `migrated_vm.mgmt.state` for both the providers? It should give you the same value.

Reply:
Nice catch :)